packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/ExpoUpdatesUpdate.kt

```diff
@@ -29,13 +29,17 @@ class ExpoUpdatesUpdate private constructor(
   private val runtimeVersion: String,
   private val launchAsset: JSONObject,
   private val assets: JSONArray?,
-  private val extensions: JSONObject?
+  private val extensions: JSONObject?,
+  private val url: Uri?,
+  private val requestHeaders: Map<String, String>?
```

Comment:
can make these non-nullable

```suggestion
  private val url: Uri,
  private val requestHeaders: Map<String, String>
```